test/test_atomic_features.py

```diff
@@ -16,9 +16,9 @@ def test_atomic_haddock():
         # - comment the first line (E0=1)
```

Comment:
Is this test just to see if the code functions?

Reply:
Yes, all this tests is that the NeuralNet.train method doesn't throw an error. The test doesn't look at the actual results.

Do you wish to extend this test somehow?

Reply:
I did not create this test, I just changed the path names so that the test will run properly from the root directory.

It seems to run the AtomicFeature class and it does some assertions on the output. It checks whether the output values are close to the expected values.

Reply:
Oh alright. I see it does assertions on the output